samples/snippets/test_sample_create_vm.py

```diff
@@ -89,46 +92,13 @@ def image(request, src_disk):
     op = image_client.insert(project=PROJECT, image_resource=image)
 
     wait_for_operation(op, PROJECT)
-
-    image = image_client.get(project=PROJECT, image=image.name)
-    request.cls.image = image
-    yield image
-
-    op = image_client.delete(project=PROJECT, image=image.name)
-    wait_for_operation(op, PROJECT)
-
-
-@pytest.fixture()
-def subnetwork():
-    network_client = compute_v1.NetworksClient()
-    network = compute_v1.Network()
-    network.name = "test-network-" + uuid.uuid4().hex[:10]
-    network.auto_create_subnetworks = True
-    op = network_client.insert(project=PROJECT, network_resource=network)
-    wait_for_operation(op, PROJECT)
-    network = network_client.get(project=PROJECT, network=network.name)
-
-    subnet = compute_v1.Subnetwork()
-    subnet.name = "test-subnet-" + uuid.uuid4().hex[:10]
-    subnet.network = network_client.get(project=PROJECT, network=network.name).self_link
-    subnet.region = "europe-central2"
-    subnet.ip_cidr_range = "10.0.0.0/20"
-    subnet_client = compute_v1.SubnetworksClient()
-    op = subnet_client.insert(
-        project=PROJECT, region="europe-central2", subnetwork_resource=subnet
-    )
-    wait_for_operation(op, PROJECT)
-    subnet = subnet_client.get(
-        project=PROJECT, region="europe-central2", subnetwork=subnet.name
-    )
-
-    yield subnet
-
-    op = subnet_client.delete(project=PROJECT, region='europe-central2', subnetwork=subnet.name)
-    wait_for_operation(op, PROJECT)
-
-    op = network_client.delete(project=PROJECT, network=network.name)
-    wait_for_operation(op, PROJECT)
+    try:
+        image = image_client.get(project=PROJECT, image=image.name)
+        request.cls.image = image
```

Comment:
@m-strzelczyk Non-blocking question: What's the benefit of using fixtures at the `class` scope instead of at the `module` scope for these tests?

Reply:
It's a bit faster. I don't create a new disk/image/snapshot for every test. As those are used in read-only way I can share them between the tests.

Reply:
Ah, sorry, didn't notice you mentioned `module` scope. This would work as well, I just picked the class scope because I found the example for it :P